liquidprompt

```diff
@@ -2343,6 +2348,28 @@ _lp_runtime_color() {
     lp_runtime_color="${LP_COLOR_RUNTIME}${lp_runtime_format}${NO_COL}"
 }
 
+######################
+# Kubernetes context #
+######################
+
+_lp_kube_context() {
+   (( LP_ENABLE_KUBECONTEXT )) || return
+
+   local _LP_KUBE_CONTEXT=$(kubectl config current-context 2>/dev/null)
+   if [[ "$LP_DELIMITER_KUBECONTEXT" ]]; then
+     lp_kube_context="${_LP_KUBE_CONTEXT%%${LP_DELIMITER_KUBECONTEXT}*}"
+   else
+     lp_kube_context="$_LP_KUBE_CONTEXT"
+   fi
+   [[ -n "$lp_kube_context" ]]
+}
+
+_lp_kube_context_color() {
+    _lp_kube_context || return "$?"
+
+    lp_kube_context_color="${LP_COLOR_KUBECONTEXT}${LP_BRACKET_OPEN}${LP_MARK_KUBECONTEXT}${lp_kube_context}${LP_BRACKET_CLOSE}${NO_COL}"
```

Comment:
Our other Development Environment theme functions are using hard coded "[]" (see python_env_color():
https://github.com/nojhan/liquidprompt/blob/2daf0e453039950d79dcf0eec5a04931e363a92a/liquidprompt#L1126

While I would be open to a PR adding a config option for those brackets, they are different than [`LP_MARK_BRACKET_OPEN`](https://liquidprompt.readthedocs.io/en/v2.0.0-rc.1/config.html#LP_MARK_BRACKET_OPEN), which the docs specify are for the central part of the prompt.

Not to mention that if in a multiplexer, the color added to these vars would break the color here:
https://github.com/nojhan/liquidprompt/blob/2daf0e453039950d79dcf0eec5a04931e363a92a/liquidprompt#L2607-L2609

Use hardcoded brackets here (for now):
```suggestion
    lp_kube_context_color="${LP_COLOR_KUBECONTEXT}[${LP_MARK_KUBECONTEXT}${lp_kube_context}]${NO_COL}"
```

Reply:
Wow your review was even faster than my question about it :-)